nodes/number/float_math.py

```diff
@@ -16,6 +16,7 @@
     ("ARCSINE", "Arcsine", "asin A", "", 7),
     ("ARCCOSINE", "Arccosine", "acos A", "", 8),
     ("ARCTANGENT", "Arctangent", "atan A", "", 9),
+
```

Comment:
actually, I had others after 3 and 18 to separate more visually (the list is soo long)
I can remove it if u think is useless, 
or add the others (to split in math/trigo/ etc)